### PR TITLE
Fix front-end linting errors (3)

### PR DESCRIPTION
**Pull request by axelboc**
I'm disabling a bunch of rules, either globally or locally. The priority is for the linting to pass so we can avoid adding new issues to the codebase.

There are a few issues left that seem to point to actual bugs or dead code. Since I'm not sure which it is, I've left them in. Perhaps @marcus-oscarsson you can take over this PR at some point and resolve those last few issues before merging?

After that, the next step will be to remove the `incrementalAdoption` flag from `.eslintrc.js` so that warnings become errors again (but there shouldn't any left) and also to enable a bunch more rules as warnings (like `no-useless-rename`, which can automatically turn `const { foo: foo } = props` into `const { foo } = props` :grin:)

### CODE REVIEW

File: ui/src/components/SampleView/DrawGridPlugin.js [hunk 1]
```diff
@@ -300,7 +300,9 @@ export default class DrawGridPlugin {
   initializeCellFilling(gd, col, row) {
     const level = this.overlayLevel ? this.overlayLevel : 0.2;
     const fill = `rgba(0, 0, 200, ${level}`;
-    return new Array(col).fill().map(() => new Array(row).fill(fill));
```
Comment (axelboc) [inline ui/src/components/SampleView/DrawGridPlugin.js:303]: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v46.0.0/docs/rules/no-new-array.md

File: ui/src/components/SampleGrid/TaskItem.jsx [hunk 2]
```diff
@@ -174,23 +174,23 @@ export class TaskItem extends React.Component {
           <div className="row" style={{ paddingTop: '0.5em' }}>
             <span className="col-sm-4">
               <b>Quality Indictor: </b>
-              <img ref="fimage" alt="First" src={qIndUrl} width="90%" />
```
Comment (axelboc) [inline ui/src/components/SampleGrid/TaskItem.jsx:177]: Couldn't find any use of these refs, but they could be hiding... Please let me know if that's the case.

**Comment by marcus-oscarsson** [comment]
Nice, thanks for the initiative ! Yepp, ill take a look !

File: ui/src/containers/SampleGridTableContainer.jsx [hunk 3]
```diff
@@ -276,7 +277,7 @@ class SampleGridTableContainer extends React.Component {
 
     const selected = this.getSamplesList()
       .filter((sampleItem) => {
-        const sampleElement = document.getElementById(sampleItem.key);
+        const sampleElement = document.querySelector(`#${sampleItem.key}`);
```
Comment (jbflo) [inline ui/src/containers/SampleGridTableContainer.jsx:280]: Hey @axelboc 

Is this linting fix work for you cause I got this:
image

to reproduce : try to select some sample in the Sample List View

Comment (axelboc) [reply ui/src/containers/SampleGridTableContainer.jsx:280]: Nice catch, indeed the column is causing an issue. I'll revert.

**Pull request merged**